docs/tasks/deploying-functions.md

```diff
@@ -51,23 +54,25 @@ def my_entry_point(context, event):
 <a id="deploying-a-simple-function"></a>
 ## Deploying a simple function
 
-To convert source code to a running function, you must first _deploy_ the function. A deployment process has three stages:
+To convert source code to a running function, you must first _deploy_ the function.
+A deployment process has three stages:
 
-1. The source code is built to a container image and pushed to a Docker registry
-2. A function object is created in Nuclio (i.e., in Kubernetes, this is a function CRD)
-3. A controller creates the appropriate function resources on the cluster (i.e., in Kubernetes this is the deployment, service, ingress, etc.)
+1. The source code is built to a container image and pushed to a Docker Registry.
+2. A function object is created in Nuclio; (in Kubernetes, this is a function CRD).
+3. A controller creates the appropriate function resources on the cluster; (in Kubernetes, this is the deployment, service, ingress, etc.).
```

Comment:
either `;` and a new sentence or parantheses
Otherwise there's a "comment" in parantheses without a context of any sentence.
I suggest to omit the parentheses in this case